spectrometer.cabal

```diff
@@ -344,6 +343,9 @@ library
     Strategy.Dart.PubDeps
     Strategy.Dart.PubSpec
     Strategy.Dart.PubSpecLock
+    Strategy.Dpkg.Dpkg
+    Strategy.Dpkg.Parser
+    Strategy.DpkgDatabase
```

Comment:
I'd recommend nesting these so you have a `Strategy.Dpkg`, `Strategy.Dpkg.Parser`, and `Strategy.Dpkg.Database`.